libs/blocks/merch-card/merch-card.js

```diff
@@ -201,6 +201,35 @@ const parseContent = async (el, merchCard) => {
       }
       return;
     }
+    if (tagName === 'H6' && element.firstElementChild?.tagName === 'EM') {
+      const calloutContentWrapper = createTag('div');
+      const calloutContent = createTag('div');
+      const emElement = element.firstElementChild;
+      let imgElement = null;
+      const children = Array.from(emElement.childNodes);
+      for (let i = 0; i < children.length; i += 1) {
+        const child = children[i];
+        if (child.nodeType === Node.ELEMENT_NODE && child.tagName === 'A' && child.innerText.trim() === '#ICON') {
+          const [imgSrc, tooltipText] = child.getAttribute('href')?.split('#') || [];
+          imgElement = document.createElement('img');
+          imgElement.src = imgSrc;
+          imgElement.title = decodeURIComponent(tooltipText);
+          imgElement.className = 'callout-icon';
+          child.parentNode.removeChild(child);
+          calloutContentWrapper.classList.add('callout-content-wrapper-with-icon');
+        } else {
+          calloutContent.append(child.cloneNode(true));
+        }
+      }
+      calloutContentWrapper.append(calloutContent);
+      if (imgElement) {
```

Comment:
no need for `{`

Reply:
Curly braces improves readability and is recommended https://stackoverflow.com/questions/4797286/are-braces-necessary-in-one-line-statements-in-javascript 
If you still feel I must omit it, let me know.

Reply:
we don't use curly braces when there's a single declaration or execution, but since you have updated the code, curly braces are needed now.